.storybook/recipes/Filters/Filters.tsx

```diff
@@ -1,31 +1,43 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import FiltersDrawer from '../../../src/components/FiltersDrawer';
 import {
   FiltersPopover,
   FiltersPopoverProps,
 } from '../../../src/components/FiltersPopover/FiltersPopover';
 
+import breakpoint from '../../../src/design-tokens/tier-1-definitions/breakpoints';
 type Props = {
-  /**
-   * Filters component to render.
-   */
-  as?: 'FiltersDrawer' | 'FiltersPopover';
   /**
    * Popover placement options relative to the filters trigger button.
    */
   placement?: FiltersPopoverProps['placement'];
 } & FiltersPopoverProps;
 
 /**
- * Primary UI component for user interaction
+ * Demonstrates usage of both FiltersPopover and FiltersDrawer depending on screensize.
  */
-export const Filters = ({ as, placement, ...other }: Props) => {
+export const Filters = ({ placement, ...other }: Props) => {
+  const [isLarge, setIsLarge] = useState(false);
+  const popoverBreakpoint = parseInt(breakpoint['eds-bp-md']) * 16; /* 1 */
```

Comment:
Do we need the `/* 1 */` on this line?
```suggestion
  const popoverBreakpoint = parseInt(breakpoint['eds-bp-md']) * 16;
```

Reply:
We do not, thanks for checking